lib/reqTemplate.js

```diff
@@ -22,6 +22,8 @@ var compilerOptions = {
 };
 
 function compileExpression(expression, part) {
+    // Convert the expression to a single line & remove whitespace.
+    expression = expression.replace(/\n/g, ' ').trim();
```

Comment:
> Wouldn't it be better to remove extra spaces after the new line as well?

Those spaces are well supported & ignored, so there is no need to remove them.

> Windows compatibility

Right now, we don't support Windows newlines at all. If we wanted to add support for that, I'd propose to normalize newlines before even parsing the yaml input.